lib/importer/disease_ontology_mirror.rb

```diff
@@ -35,8 +35,14 @@ def create_object_from_entry(entry)
       name = entry['name']
       doid = parse_doid(entry['id'])
       synonyms = process_synonyms(entry['synonym'])
-      disease = ::Disease.where(doid: doid).first_or_initialize
+      disease = if ( d = ::Disease.find_by(doid: doid) )
+                  d
+                elsif ( d = ::Disease.find_by(name: name) )
+                  d
+                else ( d = ::Disease.where(doid: doid).first_or_initialize)
+                  d
```

Comment:
I think you're missing an `end` here.

Reply:
Sigh. That's what happens when you move from Python to Ruby. Fixed.